apis/keda/v1alpha1/triggerauthentication_types.go

```diff
@@ -140,6 +139,8 @@ type AuthPodIdentity struct {
 	Provider PodIdentityProvider `json:"provider"`
 	// +optional
 	IdentityID *string `json:"identityId"`
+	// +optional
+	RoleArn string `json:"roleArn"`
```

Comment:
Do you need this for your PR? If yes, I think that we should wait until this [other PR](https://github.com/kedacore/keda/pull/5061) is merged (I plan to work on it this weekend and definitively it should be ready before the end of the year). 
I said this because I'd like to be consistent with the approach decided for that PR and not have 2 on-going PRs doing the same

Reply:
Yes I need it to be able to specify if am using workload or keda on roleArn

Reply:
okey, would you agree with waiting until my PR is ready? I'll try to push it as much as possible

Reply:
Alright no problem i will wait.